modules/appeals_api/spec/docs/supplemental_claims/v0_spec.rb

```diff
@@ -273,49 +269,19 @@ def swagger_doc
 
       response '400', 'Bad Request' do
         let(:sc_uuid) { nil }
-
-        schema type: :object,
```

Comment:
I remember seeing a while ago that this particular example didn't use the `errorModel` schema for its response, but I wasn't sure why and skipped it at the time since it didn't seem urgent. 

But it looks like the error does match the schema, so I changed this to keep things consistent